### PR TITLE
add openmodelica-ma-newsletter-2020-march

### PR DESCRIPTION
**Pull request by arun3688**

### CODE REVIEW

File: _2020-03/OpenModelica 1.15.0_Beta1_Release.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Its long-term development is supported by a non-profit organization – the Open Source Modelica Consortium [OSMC](https://www.openmodelica.org/)
+with many company, institute, and university members.
+
+![](PumpingSystem.png)
+
+The figure shows OpenModelica while editing pump component inside PumpingSystem.
+
+The OpenModelica 1.15.0 Beta1 release has been made available for testing. The main enhancement in this release is the graphical user interface support of replaceable items in libraries. The speed of the OMEdit GUI has also increased in this release. 
+
+This release is also using a further improved version of the OpenModelica Compiler New Frontend (NF), which on the average gives a factor of 10-20 speed improvement in the flattening phase of compilation. The new frontend is default also in this release.
```
Comment (beutlich) [inline _2020-03/OpenModelica 1.15.0_Beta1_Release.md]: Would be interesting to know what the base-line for the comparison was (probably the Old Frontend) and how much the flattening phase accounts for the overall translation of the frontend (on average).

File: _2020-03/OpenModelicaModProdWorkshop2020.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: MODPROD Workshop 2020	
+author: "Peter Fritzson [Open Source Modelica Consortium](https://www.openmodelica.org/)"
+category: "vendor"
+---
+
+The 14th international MODPROD Workshop on Model-Based Cyber-Physical Product Development, organized by the MODPROD competence center at Linköping University, was successfully held atLinköping University, Linköping, Sweden, February 4-5, 2020.
```
Comment (beutlich) [inline _2020-03/OpenModelicaModProdWorkshop2020.md]:
```suggestion
The 14th international MODPROD Workshop on Model-Based Cyber-Physical Product Development, organized by the MODPROD competence center at Linköping University, was successfully held at Linköping University, Linköping, Sweden, February 4-5, 2020.
```

File: _2020-03/OpenModelicaWorkShop-2020.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+title: OpenModelica Annual Workshop 2020		
+author: "Peter Fritzson [Open Source Modelica Consortium](https://www.openmodelica.org/)"
+category: "vendor"
+---
+
+The 12th OpenModelica Annual Workshop organized by the Open Source Modelica Consortium was held successfully in Linköping, Sweden, on February 3, 2020.New results and applications regarding the OpenModelica platform were presented, including status and directions of OpenModelica, modeling and simulation of innovative power generation, Modelica in the digital world, efficient minimal tearing of hybrid algebraic loops, use of DAEMode for simulation of industrial power generation system simulation, generation of symbolic Hessian matrices, virtual automation lab using Unity 3D, current technical status of OpenModelica, experiments with a prototype OpenModelica compiler in Julia, simulation of switching dynamics of electromagnets, FMI and SSP developments, sensitivity analysis and non-linear optimization with OMSens, a library for learning power systems modeling.
```
Comment (beutlich) [inline _2020-03/OpenModelicaWorkShop-2020.md]:
```suggestion
The 12th OpenModelica Annual Workshop organized by the Open Source Modelica Consortium was held successfully in Linköping, Sweden, on February 3, 2020. New results and applications regarding the OpenModelica platform were presented, including status and directions of OpenModelica, modeling and simulation of innovative power generation, Modelica in the digital world, efficient minimal tearing of hybrid algebraic loops, use of DAEMode for simulation of industrial power generation system simulation, generation of symbolic Hessian matrices, virtual automation lab using Unity 3D, current technical status of OpenModelica, experiments with a prototype OpenModelica compiler in Julia, simulation of switching dynamics of electromagnets, FMI and SSP developments, sensitivity analysis and non-linear optimization with OMSens, a library for learning power systems modeling.
```

File: _2020-03/OpenModelica 1.15.0_Beta1_Release.md (hunk 1 above)
Comment (adrpo) [reply _2020-03/OpenModelica 1.15.0_Beta1_Release.md]: Yes, the comparison is with the old frontend. The flattening phase is about 98% of the frontend, the other 2% is conversion from the new frontend structure to the old frontend structure (DAE).

**Pull request merged**